source/2019-03-30-the-ember-times-issue-91.md

```diff
@@ -53,8 +53,15 @@ This video is a nice introduction into the world of Ember Octane and Glimmer com
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Empress: the Ember Press Episode 129](https://emberweekend.com/episodes/empress-the-ember-press)
+
+Episode 129 of the Ember press featured [@mansona](https://github.com/mansona) the creator of Authmaker. This episode included a range of topics including the migration of moving the Ember guides from Middle man to be powered by Ember. Other areas include Middleman shortcomings and the rework of the guides.
```

Comment:
I think it is episode 129 of the EmberWeekend podcast, and the subject is Ember Press/Empress.
Secondly, you write Middle man with and without space between them :)